includes/functions.php

```diff
@@ -22,9 +28,9 @@ function _wpbdp_page_lookup_query( $page_id, $count = false ) {
     }
 
     if ( $count ) {
-        $query = "SELECT COUNT(*) FROM {$wpdb->posts} WHERE post_type = 'page' AND post_status = 'publish' AND ( 1=0";
+        $query = "SELECT COUNT(*) FROM {$wpdb->posts} WHERE post_type = 'page' AND post_status IN ( 'publish', 'private' ) AND ( 1=0";
```

Comment:
I don't think we need the post status check here at all. It should show if previewing a draft too.